### DISCUSSION TITLE
ترجمة  reactivity

### DISCUSSION THREAD

**Discussion by boussadjra** [vuejs-translations/docs-ar#135] [مصطلحات] [answered]
مصطلح الـReactivity هو أحد أكبر ميزات Vuejs وله واجهة برمجية خاصة تُكوّن وحدة من الوحدات الأساسية لـVue جنبا إلى جنب مع وحدة الـRenderer  ووحدة الـCompiler، خلال عملية التعريب استخدمت مصطلح "التفاعلية"  كإسم للمفهوم و "الواجهة التفاعلية" للدلالة على Reactivity API. 
بالنسبة للصفة reactive فقد استخدمت للدلالة عليها كلمة "تفاعلي" مثلا حالة تفاعلية لترجمة reactive state.
ما رأيكم؟

**Comment by Abdelaziz18003** [comment]
أظن كلمة التفاعلية هي أقرب كلمة معبرة عن reactivity، أرى أن خيارك الأول كان موفقا. والصفة المشتقة منها أيضا جيدة. الملاحظة الوحيدة لدي على ما ذكرت هي في Reactivity API، أرى أن "الواجهة التفاعلية" ليست ترجمة دقيقة ل "Reactivity API"، لأن التفاعلية هنا إسم وليست صفة للواجهة، الترجمة توحي بأن الواجهة البرمجية هي التي تملك خاصية التفاعلية، وليس العكس إذ المقصود هو أن نظام التفاعلية هو من يملك واجهة برمجية. كانت الترجمة ستكون دقيقة لوكان المصطلح "Reactive API".

إليك بعض المقترحات:

- "**واجهة التفاعلية البرمجية**" ويمكن أن تختصر حين تدرج على اللسان إلى "واجهة التفاعلية"
- "**واجهة نظام التفاعلية**": هنا ندخل كلمة نظام حتى يفهم أننا نتحدث عن نظام التفاعلية نفسه ولا داعي لذكر "البرمجة" لأنها مفهومة من السياق
- "**واجهة النظام التفاعلي**": أرى أننا حين نتحدث عن النظام فلا بأس من وصفه بالتفاعلية، كقولنا مثلا: "برنامج حاسوب" أو "برنامج حاسوبي"

**Reply by boussadjra** [reply]
ملاحظة مهمة لم انتبه إلى تفاصيلها، بالتالي يمكننا الاتفاق على الترجمة "واجهة التفاعلية" لأنها أكثر سلاسة

**Reply by Abdelaziz18003** [reply]
حسنا، أظننا متفقين على "واجهة التفاعلية" فهي مختصرة وتعبر عن المعنى

**Reply by boussadjra** [reply]
نعم، شخصيا لا أرى احتمال وجود ترجمة أفضل.

**Comment by Abdelaziz18003** [comment, marked as answer]
المصطلحات المتفق عليها في هذه المناقشة:

- مصطلح "**Reactivity**" يترجم إلى "**تفاعلية**"
- مصطلح "**Reactivity API**" يترجم إلى "**واجهة التفاعلية**"
- مصطلح "**Reactive**" يترجم إلى "**تفاعلي**"
